samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/device_tests/DeviceAccessibilitySampleTest.kt

```diff
@@ -35,7 +34,7 @@ class DeviceAccessibilitySampleTest : TestCase() {
     @get:Rule
     val activityRule = ActivityTestRule(DeviceSampleActivity::class.java, false, true)
 
-    @Test
+    // @Test
```

Comment:
Did you forgot to remove it?

Reply:
Nope, I forgot to leave a comment. I'm creating a CI for running UI tests for kaspresso PRs and unstable test is not ideal. I've tried increasing timeouts up to minute, but no luck.

Reply:
Maybe `@Ignore("FIXME")` then? :)

Reply:
Sure, it's better